multimodal/src/autogluon/multimodal/data/process_text.py

```diff
@@ -420,14 +428,21 @@ def get_pretrained_tokenizer(
             The tokenizer type, e.g., "bert", "clip", "electra", and "hf_auto".
         checkpoint_name
             Name of a pre-trained checkpoint.
+        use_fast
+            Use a fast Rust-based tokenizer if it is supported for a given model.
+            If a fast tokenizer is not available for a given model, a normal Python-based tokenizer is returned instead.
+            See: https://huggingface.co/docs/transformers/model_doc/auto#transformers.AutoTokenizer.from_pretrained.use_fast
 
         Returns
         -------
         A tokenizer instance.
         """
         try:
             tokenizer_class = ALL_TOKENIZERS[tokenizer_name]
-            return tokenizer_class.from_pretrained(checkpoint_name)
+            if use_fast is None:  # to be backward compatible
```

Comment:
Can we default it to True for backward compatibility?

Reply:
Done